packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -170,7 +170,10 @@ export class SwapQuoter {
         this._devUtilsContract = new DevUtilsContract(this._contractAddresses.devUtils, provider);
         this._protocolFeeUtils = new ProtocolFeeUtils(constants.PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS);
         this._orderStateUtils = new OrderStateUtils(this._devUtilsContract);
-        this._quoteRequestor = new QuoteRequestor(options.rfqt ? options.rfqt.makerEndpoints || [] : []);
+        this._quoteRequestor = new QuoteRequestor(
+            options.rfqt ? options.rfqt.makerEndpoints || [] : [],
+            options.rfqt ? options.rfqt.warningLogger || undefined : undefined,
```

Comment:
nit but can't this just be `options.rfqt ? options.rfqt.warningLogger : undefined`?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2562/commits/22f9b03fced5c035d28f16995a342c58d93f512c